### PR TITLE
fix: comment false does not work

### PR DESCRIPTION
**Pull request by faizplus**
Fixes: #427 
Replaced `getInput` to `getBooleanInput` for reading boolean options

### CODE REVIEW

**Review by simoneb — APPROVED** [review]

**Comment by faizplus** [comment]
@simoneb I can't merge this. Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

**Pull request merged**
